eureka-client/src/main/java/com/netflix/discovery/shared/transport/jersey/JerseyEurekaHttpClientFactory.java

```diff
@@ -153,7 +162,7 @@ private JerseyEurekaHttpClientFactory buildLegacy() {
                     .withReadTimeout(readTimeout)
                     .withMaxConnectionsPerHost(maxConnectionsPerHost)
                     .withMaxTotalConnections(maxTotalConnections)
-                    .withConnectionIdleTimeout(connectionIdleTimeout)
+                    .withConnectionIdleTimeout((int) connectionIdleTimeout)
```

Comment:
We use consistently long for any time related properties. Here we map to legacy that will be removed in the future.